### PR TITLE
Fixed: Logic to recieveAll returns on return-details page(#2t2wcjr)

### PR DESCRIPTION
**Pull request by shashwatbangar**
### Related Issues

Closes #

### Short Description and Why It's Useful

When we want to receive all return shipments, we always will get one item, so its unnecessary to use filter and loop over whole current items array. So we will use find here so as soon as we meet the condition, it returns without looping over array.

### Screenshots of Visual Changes before/after (If There Are Any)

**IMPORTANT NOTICE** - Remember to add changelog entry

### Contribution and Currently Important Rules Acceptance

- [ ] I read and followed [contribution rules](https://github.com/hotwax/receiving#contribution-guideline)

### CODE REVIEW

File: src/views/ReturnDetails.vue [hunk 1]
```diff
@@ -175,15 +175,15 @@ export default defineComponent({
       return alert.present();
     },
     async receiveReturn() {
-      await this.store.dispatch('return/receiveReturn', {payload: this.current}).then(() => {
-        this.router.push('/returns');
-      })   
+      await this.store.dispatch('return/receiveReturn', {payload: this.current});
```
Comment (adityasharma7) [inline src/views/ReturnDetails.vue:178]: I think we should handle for the error

File: src/views/ReturnDetails.vue [hunk 2]
```diff
@@ -175,15 +175,15 @@ export default defineComponent({
...
+      this.router.push('/returns');
     },
     receiveAll(item: any) {
-      this.current.items.filter((ele: any) => {
+      this.current.items.find((ele: any) => {
```
Comment (adityasharma7) [inline src/views/ReturnDetails.vue:182]: I think we should find the element and store it in a variable, then update it

**Review by adityasharma7 — CHANGES_REQUESTED** [review]

File: src/views/ReturnDetails.vue (hunk 1 above)
Comment (shashwatbangar) [reply src/views/ReturnDetails.vue:178]: done sir

File: src/views/ReturnDetails.vue [hunk 3]
```diff
@@ -175,15 +176,17 @@ export default defineComponent({
...
-      })   
+      }
     },
     receiveAll(item: any) {
-      this.current.items.filter((ele: any) => {
+      let element = this.current.items.find((ele: any) => {
         if(ele.itemSeqId == item.itemSeqId) {
-          ele.quantityAccepted = ele.quantityOrdered;
-          ele.progress = ele.quantityAccepted / ele.quantityOrdered
+          element.quantityAccepted = ele.quantityOrdered;
```
Comment (adityasharma7) [inline src/views/ReturnDetails.vue:187]: I think this will not work, we are updating element which is not defined

**Review by adityasharma7 — CHANGES_REQUESTED** [review]

**Pull request merged**